src/sisyphus/core.clj

```diff
@@ -134,10 +135,20 @@
     (update state :kafka merge consumer)))
 
 (defn start!
-  "Start the system by making all the required connections and returning the state map."
+  "Start the system by making all the required connections and returning the state map.
+   The priority of values for rabbit config are
+     1. command line options
+     2. gce metadata fields
+     3. values from file resources/config/sisyphus.clj
+     4. defaults from ns rabbit/default-config"
```

Comment:
> and then sisyphus/rabbit code derives the 3 rabbit queuing parameters from that workflow name?

Ah I like this, this is the way to go. Yeah let's just supply one parameter and sisyphus can work out the details. 

> Is gce-metadata not getting results due to not running on a GCE VM? I tested it on a GCE worker node.

Ah yeah, maybe doesn't work from a remote machine? I am leaning to doing this with the compute service from the previous PR instead of an HTTP call where you have to worry about passing in credentials etc if you are not in the network. I'll look into it.

Reply:
I updated `gce-metadata` to just return nil if it doesn't find anything, then you can use `or` to provide a default. There was a case where I want to do something if it failed, not read a default value, and this is simpler too. 

With that Sisyphus builds the various rabbit configuration values ("exchange", "queue", "routing-key") from a single metadata parameter: `attributes/workflow`. 

Thanks for all the feedback on this one!